scripts/ci-checks.sh

```diff
@@ -109,9 +109,9 @@ endgroup
 
 group "Python format"
 if [ $FIX -ne 0 ]; then
-  git ls-files tests/ python/ scripts/ .cmake-format.py | grep -e '\.py$' | xargs black
+  git ls-files tests/ python/ scripts/ tla/ .cmake-format.py | grep -e '\.py$' | xargs black
```

Comment:
We missed the formatting in that directory, this leads to a bit of one-time noise on this PR unfortunately.

Reply:
Noted, in an ideal world this would be a separate PR

Reply:
Agreed

Reply:
Fixed now